ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -134,6 +127,43 @@ export default class AddContact extends PureComponent {
               {t('ethereumPublicAddress')}
             </div>
             {this.renderInput()}
+            <div
+              className={`address-book__view-contact__group__${
+                domainResolutions?.length === 1 ? 'single-' : ''
+              }resolution-list`}
+            >
+              {domainResolutions?.length > 0 &&
+                domainResolutions.map((resolution) => {
```

Comment:
Can we remove the `domainResolutions?.length` check and just do `domainResolutions?.map`?  Mapping over empty would be fine.